pkgs/development/python-modules/pytest-workflow/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  jsonschema,
+  pytest,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-workflow";
+  version = "2.0.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "LUMC";
+    repo = "pytest-workflow";
+    rev = "v${version}";
+    hash = "sha256-ztR4TW41qVAnCLXPUsWaiMuD1ESL0Kvd5KAsOf8tcE0=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    jsonschema
+    pytest
+    pyyaml
+  ];
+
+  pythonImportsCheck = [ "pytest_workflow" ];
+
+  meta = with lib; {
+    description = "Configure workflow/pipeline tests using yaml files";
+    homepage = "https://github.com/LUMC/pytest-workflow";
+    license = licenses.agpl3Only;
```

```suggestion
    license = licenses.agpl3Plus;
```
see https://github.com/LUMC/pytest-workflow/blob/18eb79c3b489ab2349ffa2468e6af6d9966b100e/setup.py#L4-L7
and https://github.com/LUMC/pytest-workflow/blob/18eb79c3b489ab2349ffa2468e6af6d9966b100e/setup.py#L49

---

Great catch! That was just what `nix-init` pulled in.